src/components/v5/shared/Button/Button.tsx

```diff
@@ -13,6 +12,7 @@ const displayName = 'v5.Button';
 
```

Comment:
v5.shared.Button

Reply:
As I can see, we have never used the prefix `shared `for components inside this directory.